wgpu-types/src/lib.rs

```diff
@@ -26,6 +27,10 @@ pub const COPY_BYTES_PER_ROW_ALIGNMENT: u32 = 256;
 pub const BIND_BUFFER_ALIGNMENT: BufferAddress = 256;
 /// Buffer to buffer copy offsets and sizes must be aligned to this number.
 pub const COPY_BUFFER_ALIGNMENT: BufferAddress = 4;
+/// Total amount of possible shader stages.
+pub const SHADER_STAGE_COUNT: usize = 3;
```

Comment:
Does this need to be public in `wgpu-types`? Maybe it should just be private in `wgpu-core` instead?

Reply:
Nope, will move it.

Reply:
still an issue?

Reply:
Forgot to remove the old one.

Reply:
still in wgpu-types?